README.md

```diff
@@ -30,3 +30,91 @@ TODO :
   present is an update from the v130 master branch only).
 - Do something about the 3rd party initiatives to provide some Docker or other 
   Plug and Play versions.
+
+## Installation
+
+[Download](https://github.com/oscarpilote/Ortho4XP/archive/refs/heads/master.zip) this repository and extract it to a preferred location on your computer and follow the instructions for your operation system below.
+
+### Windows
+
+1. Download and install [Python 3.11.9](https://www.python.org/downloads/release/python-3119/). Use the "Customize installation" option and make sure "pip" and "tcl/tk and IDLE" are selected on the Optional Features page.
+
+2. From a Command Prompt window, install the required packages:
+```py -3.11 -m pip install -r requirements.txt```
+
+3. Go to the Ortho4XP folder:
+```cd /path/to/Ortho4XP```
+
+4. Launch Ortho4XP:
+```py -3.11 -m Ortho4XP_v130.py```
+
+#### (Optional) Install tools needed for using elevation data (custom_dem) other than the default:
+
+1. Download and install the latest [Microsoft Visual C++ Redistributable (Microsoft C and C++ (MSVC) runtime libraries)](https://aka.ms/vs/17/release/vc_redist.x86.exe). The link provided is specific for Windowws 64-bit (X86). If you need a different version, they can be found [here](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170#latest-microsoft-visual-c-redistributable-version).
+
+2. Download the [Geospatial library wheels for Python](https://github.com/cgohlke/geospatial-wheels/releases/download/v2024.2.18/GDAL-3.8.4-cp311-cp311-win_amd64.whl). The link provided is specific for Python 3.11 and Windows 64-bit (X86). If you need a different version, they can be found [here](https://github.com/cgohlke/geospatial-wheels/releases).
+
+3. Install GDAL (update the path to where you downloaded the file in step 2):
+```py -3.11 -m "/path_to_downloaded_file_in_step_2/GDAL-3.8.4-cp311-cp311-win_amd64.whl"```
```

Comment:
Tested this locally. Shouldn't this be

```
py -3.11 -m pip install "path_to_downloaded_file_in_step_2/GDAL-3.8.4-cp311-cp311-win_amd64.whl"
```

Reply:
Yeah that's my bad - I rushed through this and didn't really work on it. I'm going to post another message here with some working steps for Python 3.12.3 on Windows, as well as some more ideas.